Mathlib/Order/Hom/Set.lean

```diff
@@ -159,6 +160,16 @@ instance subsingleton_of_wellFoundedGT' [LinearOrder β] [WellFoundedGT β] [Pre
 
 instance unique_of_wellFoundedGT [LinearOrder α] [WellFoundedGT α] : Unique (α ≃o α) := Unique.mk' _
 
+/-- An order isomorphism between lattices induces an order isomorphism between corresponding
+interval sublattices. -/
+protected def Iic [Lattice α] [Lattice β] (e : α ≃o β) (x : α) :
```

Comment:
Are there similar isoms for other `Ixx`?

Reply:
Yes, I've added `Ici` and `Icc`.